index.js

```diff
@@ -59,14 +59,58 @@ animalToRemove.remove()
 /************************** EVENTS JS MINI CHALLENGE ******************************/
 
 /***** Deliverable 1 *****/
+
 function toggleColor(element) {
     if (element.style.color === "green") {
-        element.style.color = "black"
+        element.style.color = "red"
     } else {
         element.style.color = "green"
     }
 }
 
+header.addEventListener('click', function(){
+    if (header.style.color === 'green'){
+        header.style.color = "red"
+    } else {
+        header.style.color = "green"
+    }  
+})
+
 /***** Deliverable 2 *****/
 
+const button = document.querySelector('.like-button')
+button.addEventListener('click', function(){
+
+   const likes = document.querySelector('.likes')
+   likes.textContent = `${traveler.likes + 1} Likes `
```

Comment:
This works, but what if we didn't have that specific traveler's likes easily available to us and we still only wanted to update the DOM?

Here are some steps we could take

First find the like button element on the page
Then add an event listener to the like button
We are listening for a `click` event, and we will pass a callback function to be invoked when the event fires.
Then we create the function `addNewLike` which will handle all the functionality.
The `addNewLike` function has a few steps:
1. find the `p` tag where the current number of likes is stored on the page
2. extract the number out of the string, make sure to convert it to a number with `parseInt()`
3. add 1 to that number.
4. Create a new string with the new number
5. update the `p` tag to have the newly created string as its `innerText`.
```javascript
const likeBtn = document.querySelector(".like-button");
function addNewLike(e){
    const likesPTag = e.target.previousElementSibling;
    const likesText = likesPTag.innerText;
    let currentLikes = parseInt(likesText.split(" ")[0]);
    currentLikes += 1;
    const newLikesString = `${currentLikes} Likes`;
    likesPTag.innerText = newLikesString;
}
likeBtn.addEventListener("click", addNewLike)
```
With ES6 notation
```javascript
const likeBtn = document.querySelector(".like-button");
const addNewLike = e => {
    const likesPTag = e.target.previousElementSibling;
    let currentLikes = parseInt(likesPTag.innerText.split(" ")[0]);
    currentLikes += 1;
    const newLikesString = `${currentLikes} Likes`;
    likesPTag.innerText = newLikesString;
}
likeBtn.addEventListener("click", addNewLike)
```